pages/01.administrate/09.providers/03.vpn/vpn.md

```diff
@@ -0,0 +1,87 @@
+---
+title: VPN providers
+template: docs
+taxonomy:
+    category: docs
+never_cache_twig: true
+twig_first: true
+process:
+    markdown: true
+    twig: true
+routes:
+  default: '/providers/vpn'
+---
+
+Since setting up a server at home is an uncommon practice, most Internet connections provided to individuals are unsuitable for this purpose especially if you desire to send mail. A net neutral VPN providing a dedicated fixed public IPv4 address and IPv6 addresses [can help to circumvent some limitations or difficulties](/vpn_advantage).
+
+
+Below, you can find a list of providers compatible for self-hosting and especially those providing .cube format for VPNClient apps and those providing [internetcube](https://internetcu.be).
+
+!!! By **compatible for self-hosting** we means VPN offers with at least:
+!!! * a fixed dedicated public IPv4
+!!! * port forwarding or opened features
+!!! * net neutrality: no traffic analysis, no user data resale, no alteration of traffic (without legal obligations)...
+
+------------------
+
+[ui-tabs position="top-left" active="0" theme="lite"]
+[ui-tab title="English speaking-site"]
+
+| VPN provider | [VPNClient](https://github.com/labriqueinternet/vpnclient_ynh) compatibility | IPv6 | rDNS IPv4 | rDNS IPv6 | Price | Membership | Net Neutrality |
+| ------------ | ----------------------- | ---- | --------- | --------- | ----- | -------------- |
+| [Neutrinet](https://neutrinet.be/en/vpn)    | ✔ (.cube + [internetcube](https://internetcu.be))               | ✔    | ✔         | ?         | ~8 €¹ / month | included | <span class="ffdn">FFDN</span> <span class="label label-success label-as-badge">Non Profit</span> |
+¹ [Pay what you want](https://en.wikipedia.org/wiki/Pay_what_you_want)
+
+!!! If you try an other VPN provider that include **public dedicated ipv4 and port forwarding**, feel free to contribute to this documentation. We need people to test specific offers of those commercial providers:
+!!! * [VPN area](https://vpnarea.com/front/home/dedicated-ip)
+!!! * [Trust zone](https://trust.zone/fr/order?p=25)
+!!! * [PureVPN](https://www.purevpn.fr/ip-vpn-dedie)
+!!! * [RapidVPN](https://www.rapidvpn.com/vpn)
+[/ui-tab]
+[ui-tab title="French speaking-site"]
+
+| Fournisseurs | Compatibilité [VPNClient](https://github.com/labriqueinternet/vpnclient_ynh) | IPv6 | rDNS IPv4 | rDNS IPv6 | Prix | Adhésion | Neutralité du net |
+| ------------ | ----------------------- | ---- | --------- | --------- | ----- | -------------- |
+| [Aquilenet](https://www.aquilenet.fr/vpn/)    | ✔ (.cube)               | ✔    | ?         | ?         | 3 à 5 / mois | ~15€¹ /an | <span class="ffdn">FFDN</span> <span class="label label-success label-as-badge">Non Profit</span> |
```

Comment:
Yes Aquilenet provides IPv4 and IPv6 rDNS personalization.

Reply:
```suggestion
| [Aquilenet](https://www.aquilenet.fr/vpn/)    | ✔ (.cube)               | ✔    | ✔         | ✔         | 3 à 5 / mois | ~15€¹ /an | <span class="ffdn">FFDN</span> <span class="label label-success label-as-badge">Non Profit</span> |
```